lib/project_types/node/commands/create.rb

```diff
@@ -44,7 +44,9 @@ def call(args, _name)
 
         @ctx.puts(@ctx.message('apps.create.info.created', form.title, partners_url))
         @ctx.puts(@ctx.message('apps.create.info.serve', form.name, ShopifyCli::TOOL_NAME))
-        @ctx.puts(@ctx.message('apps.create.info.install', partners_url, form.title))
+        unless ShopifyCli::Shopifolk.acting_as_shopify_organization?
```

Comment:
We need this, otherwise it outputs:

`Then, visit %s/test to install %s on your Dev Store`

which is not valid for 1st party apps.